### PR TITLE
feat(logs): button update time format

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-608)

- Add time format dropdown for Deployment logs

Capture d’écran 2022-12-09 à 16 40 26

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(logs): button update time format - 2022-12-02T13:47:35Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/afdcdf40-2f6a-4f03-9d56-28472a271666/applications)

 Another comment will be posted when deployments are terminated

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://zafdcdf40-ze96be9db-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://zafdcdf40-ze47d45e9-gtw.za8ad0659.bool.sh)

File: libs/shared/ui/src/lib/components/buttons/button/button.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ export enum ButtonStyle {
   TAB = 'tab',
   ERROR = 'error',
   CONFIRM = 'confirm',
+  DARK = 'dark',
```
Comment (bdebon) [inline libs/shared/ui/src/lib/components/buttons/button/button.tsx:22]: would not it be better to not add a dark style but just add CSS tailwind helper on the normal style to convert the button in dark? It's an open discussion, maybe the DARK enum is really needed.

Comment (bdebon) [reply libs/shared/ui/src/lib/components/buttons/button/button.tsx:22]: Or maybe it's the term DARK that is not super explicit and we should maybe choose something like: CONTRASTED or NEGATIVE or sthg else? 
wdyt?

**Review by bdebon — COMMENTED** [review]
Perfect Remi, just a smal question about the DARK naming for the enum.

**Pull request merged**